### PR TITLE
feat(dashboard): connect delete api

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

SSIA

### Things to Talk About

Dashboard permissions were dummy data. My commit also.
I've added `TODO` annotation, should be discussed soon.

### CODE REVIEW

File: src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue [hunk 1]
```diff
@@ -108,7 +108,7 @@ export default defineComponent({
             observeInstance.observe(containerRef?.value as Element);
         });
 
-        onUnmounted(() => {
```
Comment (minseolee) [inline src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue:111]: This is WidgetContainer. 
When I remove a dashboard. some error has occurred.

On `onMounted`, element has removed already.
So, `onBeforeUnmount` hooks is correct .

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/services/dashboards/dashboard-detail/DashboardDetailPage.vue [hunk 2]
```diff
@@ -119,15 +118,25 @@ const handleVisibleCloneModal = () => {
...
-(async () => {
+const getDashboardData = async () => {
     try {
-        const result: ProjectDashboardModel = await SpaceConnector.clientV2.dashboard.projectDashboard.get({ project_dashboard_id: 'project-dash-eb13f8b042f9' });
+        let result: ProjectDashboardModel;
+        if (isProjectDashboard) {
+            result = await SpaceConnector.clientV2.dashboard.projectDashboard.get({ project_dashboard_id: props.dashboardId });
+        } else {
+            result = await SpaceConnector.clientV2.dashboard.domainDashboard.get({ project_dashboard_id: props.dashboardId });
+        }
```
Comment (sulmoJ) [inline src/services/dashboards/dashboard-detail/DashboardDetailPage.vue:128]: 👍 👍

**Review by sulmoJ — APPROVED** [review]
LGTM!!
Nice clean code :D

**Pull request merged**